Sources/SwiftDocC/SwiftDocC.docc/footer.html

```diff
@@ -11,7 +11,7 @@
 <footer id="footer" role="contentinfo" hidden>
   <div class="container">
     <div>
-      <p class="copyright">Copyright © 2021 Apple Inc. All rights reserved.</p>
+      <p class="copyright">Copyright © 2022 Apple Inc. All rights reserved.</p>
```

Comment:
@mportiz08 @dobromir-hristov is there a Fancy Pants solution we can use to make sure this is always up-to-date?

Reply:
Potentially you could add some JavaScript logic to do it here: https://github.com/apple/swift-org-website/blob/e904c71e375fc9b7fa676b798ef925cdb46818ee/assets/javascripts/docc/footer.js